docs/design-proposals/multi-tenancy/multi-tenancy-api-resource-model.md

```diff
@@ -159,14 +159,88 @@ Here are some examples:
 
 ### CRD Support
 
- Each tenant can install their CRD independently, without worrying about resource name collisions. The API group/vision/kind (G/V/K) defined by the CRD object is only accessible to that tenant.
+#### Principles and Outlines of Design
+
+ The following Arktos multi-tenancy design principles will be observed in the design of CRD:
  
- For CRDs installed in system world, by default it's only accessible for tenant users. But if it's marked with "multi-tenancy.k8s.io/published" annotation, it will be accessible to all tenants.
+ 1. **isolation**. Each regular tenant can independently create/delete/update their CRDs and custom resources based on the CRDs. There will be NO CRD G/V/K collision between two regular tenants. Actually, the CRDs and custom resources of one regular tenant are invisible to another regular tenant.  
+
+ 2. **Managebility**. A cluster operator (he should be user under the system tenant) can introduce a new CRD, where he can choose to:
```

Comment:
If the design of manageability part is not clear we can skip it for now.

Reply:
Add some notes that this part is still under update.